homework/07LessonSeven/script.js

```diff
@@ -69,20 +69,23 @@ $(document).on('ready', function() {
   // Problem 9:
   // select the element with id="insert-text"
   // then set the text of this element to "Text Has Been Inserted"
-
+   $("#insert-text").append("Text Has Been Inserted");
 
   // Problem 10:
   // select the element with id="add-attribute"
   // add the attribute data-color="red" to that element
-
-
+    var $add = $("#add-attribute");
+     $add.attr('data-color','red');
```

Comment:
You can also calll `attr()` directly on the `$("#add-attribute")` jquery object.